common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayLocationV2.scala

```diff
@@ -19,17 +20,19 @@ sealed trait DisplayLocationV2
 
 object DisplayLocationV2 {
   def apply(location: Location): DisplayLocationV2 = location match {
-    case l: DigitalLocation =>
+    case DigitalLocation(url, locType, license, credit, accessCondition, _) =>
       DisplayDigitalLocationV2(
-        locationType = DisplayLocationType(l.locationType),
-        url = l.url,
-        credit = l.credit,
-        license = l.license.map(DisplayLicenseV2(_))
+        locationType = DisplayLocationType(locType),
+        url = url,
+        credit = credit,
+        license = license.map(DisplayLicenseV2(_)),
+        accessCondition = accessCondition.map(DisplayAccessCondition(_))
       )
-    case l: PhysicalLocation =>
+    case PhysicalLocation(locationType, label, _) =>
       DisplayPhysicalLocationV2(
-        locationType = DisplayLocationType(l.locationType),
-        label = l.label)
+        locationType = DisplayLocationType(locationType),
+        label = label
+      )
```

Comment:
No access conditions on the display physical locations? Sounds like that's what https://github.com/wellcometrust/platform/issues/4125 is describing.